libraries/AP_Logger/AP_Logger.h

```diff
@@ -325,9 +326,11 @@ class AP_Logger
 
     void Write(const char *name, const char *labels, const char *fmt, ...);
     void Write(const char *name, const char *labels, const char *units, const char *mults, const char *fmt, ...);
+    void WriteStreaming(const char *name, const char *labels, const char *fmt, ...);
```

Comment:
If we could get some comments above the methods to clarify to the dev which one to use would be nice.